src/ompl/geometric/planners/cforest/src/CForest.cpp

```diff
@@ -75,8 +75,11 @@ void ompl::geometric::CForest::addPlannerInstanceInternal(const base::PlannerPtr
     else
     {
         planner->setProblemDefinition(pdef_);
-        if (planner->params().hasParam("prune"))
-            planner->params()["prune"] = prune_;
+        if (planner->params().hasParam("focus_search"))
+            planner->params()["focus_search"] = prune_;
+        else
+            OMPL_WARN("%s does not appear to support search focusing.", planner->getName().c_str());
+
```

Comment:
More about this change explained in the RRTstar code, but my general thought here is to turn on a generic parameter called *focus_search*, instead of something as specifically named as *prune*.

This should make it conceptually cleaner for other planners to support the CFOREST framework even if they focus their search in ways other than pruning the tree.